sdk/attestation/azure-security-attestation/inc/azure/attestation/attestation_administration_client.hpp

```diff
@@ -44,6 +44,8 @@ namespace Azure { namespace Security { namespace Attestation {
    *
    */
   class AttestationAdministrationClient final {
```

Comment:
BTW, this class doesn't need virtual destructor so far - it does not inherit anything, and is not inheritable. And if it becomes that, most likely we'll have to restrict copy constructor and assignment, so by-value return from the factory will stop working as well.